ci/bootstrap.yaml

```diff
@@ -69,3 +69,22 @@ roleRef:
   kind: ClusterRole
   name: auth-driver
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: admin
+  namespace: default
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: default-admin-cluster-admin-binding
+subjects:
+- kind: ServiceAccount
+  name: admin
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Where is this cluster role defined?

Reply:
It's a default role in k8s https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles